### PR TITLE
Add experimental features to shipping samples

### PR DESCRIPTION
**Pull request by beaufortfrancois**
With this patch, features behind the Experimental Framework will appear in samples as they will be considered as shipped.

R=@ebidel

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
(gentle ping) @ebidel

File: models.py [hunk 1]
```diff
@@ -363,6 +363,15 @@ def get_shipping_samples(self, limit=None, update_cache=False):
...
       features = q.fetch(None)
 
+      # Get all experimental framework features. Ordered by shipping milestone (latest first).
+      q = Feature.all()
+      q.filter('impl_status_chrome =', EXPERIMENTAL_FRAMEWORK)
+      q.order('-impl_status_chrome')
+      q.order('-shipped_milestone')
+      q.order('name')
+      experimental_features = q.fetch(None)
+      features.extend(experimental_features)
```
Comment (ebidel) [inline models.py]: this will put all the experimental features at the end of the list instead of interweaving them. Is that what you want?

**Comment by beaufortfrancois** [comment]
Feedback received ;)
Thank you @ebidel !

**Comment by ebidel** [comment]
Lgtm

**Comment by beaufortfrancois** [comment]
Thank you!
Can we push this to live?

**Comment by ebidel** [comment]
Deployed

**Comment by beaufortfrancois** [comment]
Merci ;)

**Pull request merged**